### PR TITLE
Update Sensors Network

### PR DESCRIPTION
**Pull request by nakata5321**
Rework all articles in Open Sensors Network part.

### CODE REVIEW

File: docs/en/sensors-connectivity-setup.md [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+wget https://dist.ipfs.io/go-ipfs/v0.8.0/go-ipfs_v0.8.0_linux-amd64.tar.gz
+tar -xzf go-ipfs_v0.8.0_linux-amd64.tar.gz
+cd go-ipfs
+sudo bash install.sh 
+ipfs init
+```
+Also, we need to install `python3 dev` package and `pip`:
+
+```shell
+sudo apt install python3-dev
```
Comment (Vourhey) [inline docs/en/sensors-connectivity-setup.md:31]: Let's merge it into a single line command

File: docs/en/configuration-options-description.md [hunk 2]
```diff
@@ -282,9 +156,31 @@ Connect you SDS011 sensor to a USB port, let's assume it got `/dev/ttyUSB0` addr
 }
 ```
 
-## Scenario #4: Connect Multiple Sensors and Publish to Datalog
+And start Sensors Connectivity module.
+
+## Scenario #3: Publish sensors data to Datalog
+
+In this scenario it doesn’t matter which sensors connection type to choose. Example will show you instructions is written for default one - over `http`.
```
[the remark this answers is not in the document]
Comment (nakata5321) [reply docs/en/configuration-options-description.md:163]: Хотел сказать, что в этом сценарии неважно как вы получаете входные данные - через mqtt, http или COM. тут мы рассматриваем именно вариант уже выхода данных из коннективити

File: docs/en/configuration-options-description.md [hunk 3]
```diff
@@ -5,108 +5,33 @@ contributors: [LoSk-p, Vourhey, tubleronchik]
...
 ---
 
-Basically, you can think of the package as a black box with one input (sensor data) and many outputs.
+Basically, you can think of the Sensors Connectivity module as a black box with one input (sensor data) and many outputs.
 For now only SDS011 sensor is supported, but if you are familiar with Python it'd be easy to add other sensors as well.
 
-Have a look at [configuration](https://github.com/airalab/sensors-connectivity/blob/master/config/default.json) file:
+At the moment it's possible to publish data to [Luftdaten](https://luftdaten.info/) and [Robonomics Network Datalog](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fkusama.rpc.robonomics.network%2F#/explorer). 
+Robonomics team prepare some ready configuration files to use.
+Full overview of configuration fields you can find [here.](https://github.com/airalab/sensors-connectivity/tree/master/connectivity/config)
```
[the remark this answers is not in the document]
Comment (nakata5321) [reply docs/en/configuration-options-description.md]: solve in https://github.com/airalab/robonomics-wiki/pull/236/commits/58285cacfd4c7308b14a9952502745f80e082cc4

File: docs/en/configuration-options-description.md [hunk 4]
```diff
@@ -5,108 +5,33 @@ contributors: [LoSk-p, Vourhey, tubleronchik]
...
-## Scenario #1: Connect SDS011 to serial port
+```bash
+$ ls -l /dev/serial/by-id
+total 0
+lrwxrwxrwx 1 root root 13 сен  5 14:01 usb-1a86_USB2.0-Ser_-if00-port0 -> ../../ttyUSB0
+```
 
-The easiest and the most straightforward way to connect your sensor to the network is using the serial port
+In the example it is `ttyUSB0`.
+Now you need to create new configuration file, or edit existing configuration file from previous [article](/docs/sensors-connectivity-setup/#json-configuration).
```
[the remark this answers is not in the document]
Comment (nakata5321) [reply docs/en/configuration-options-description.md]: solve in solve in https://github.com/airalab/robonomics-wiki/pull/236/commits/58285cacfd4c7308b14a9952502745f80e082cc4

File: docs/en/configuration-options-description.md [hunk 5]
```diff
@@ -5,108 +5,33 @@ contributors: [LoSk-p, Vourhey, tubleronchik]
...
+Insert what you see below to configuration file . Write full path to your database in file.
 
-Connect you SDS011 sensor to a USB port, let's assume it got `/dev/ttyUSB0` address
+<robo-wiki-note type="okay">
+Don't forget to insert your Board path to port statement and latitude and longitude of a sensor to geo statement.
```
[the remark this answers is not in the document]
Comment (nakata5321) [reply docs/en/configuration-options-description.md]: solve in https://github.com/airalab/robonomics-wiki/pull/236/commits/58285cacfd4c7308b14a9952502745f80e082cc4

File: docs/en/configuration-options-description.md [hunk 6]
```diff
@@ -161,74 +86,23 @@ Connect you SDS011 sensor to a USB port, let's assume it got `/dev/ttyUSB0` addr
...
->
-> On NixOS you can do:
-> ```
-> networking.firewall.allowedTCPPorts = [ 31313 ];
-> ```
+Assume that you already have MQTT broker [mosquitto](https://mosquitto.org/download/) or similar.
 
-## Scenario #3: Connect SDS011 via MQTT
+As in previous scenario you need to create new configuration file, or edit existing configuration file from previous [article](/docs/sensors-connectivity-setup/#json-configuration).
+Insert what you see below to configuration file. Write full path to your database in file.
```
[the remark this answers is not in the document]
Comment (nakata5321) [reply docs/en/configuration-options-description.md]: solve in https://github.com/airalab/robonomics-wiki/pull/236/commits/58285cacfd4c7308b14a9952502745f80e082cc4

File: docs/en/configuration-options-description.md [hunk 7]
```diff
@@ -282,9 +156,31 @@ Connect you SDS011 sensor to a USB port, let's assume it got `/dev/ttyUSB0` addr
...
-## Scenario #4: Connect Multiple Sensors and Publish to Datalog
+And start Sensors Connectivity module.
+
+## Scenario #3: Publish sensors data to Datalog
+
+In this scenario it doesn’t matter which sensors connection type to choose. Example will show you instructions is written for default one - over `http`.
+
+This scenario shows how to upload your sensor's data to Robonomics Parachain Datalog. 
+Robonomics Datalog is analog of "Telemetry" in Web3 technologies. 
+Datalog create a sensor's data snapshot each period of time, which increase reliability of data.
```
[the remark this answers is not in the document]
Comment (nakata5321) [reply docs/en/configuration-options-description.md]: solve in https://github.com/airalab/robonomics-wiki/pull/236/commits/58285cacfd4c7308b14a9952502745f80e082cc4

File: docs/en/configuration-options-description.md [hunk 8]
```diff
@@ -282,9 +156,31 @@ Connect you SDS011 sensor to a USB port, let's assume it got `/dev/ttyUSB0` addr
...
 
-### Configuration
+Just like in previous scenario you need to create new configuration file, or edit existing configuration file from previous [article](/docs/sensors-connectivity-setup/#json-configuration).
```
[the remark this answers is not in the document]
Comment (nakata5321) [reply docs/en/configuration-options-description.md]: solve in https://github.com/airalab/robonomics-wiki/pull/236/commits/58285cacfd4c7308b14a9952502745f80e082cc4

File: docs/en/configuration-options-description.md [hunk 9]
```diff
@@ -282,9 +156,31 @@ Connect you SDS011 sensor to a USB port, let's assume it got `/dev/ttyUSB0` addr
...
+
+In this scenario it doesn’t matter which sensors connection type to choose. Example will show you instructions is written for default one - over `http`.
+
+This scenario shows how to upload your sensor's data to Robonomics Parachain Datalog. 
+Robonomics Datalog is analog of "Telemetry" in Web3 technologies. 
+Datalog create a sensor's data snapshot each period of time, which increase reliability of data.
 
-### Configuration
+Just like in previous scenario you need to create new configuration file, or edit existing configuration file from previous [article](/docs/sensors-connectivity-setup/#json-configuration). 
+Insert what you see below to configuration file.  Write full path to your database in file.
```
[the remark this answers is not in the document]
Comment (nakata5321) [reply docs/en/configuration-options-description.md]: solve in https://github.com/airalab/robonomics-wiki/pull/236/commits/58285cacfd4c7308b14a9952502745f80e082cc4

File: docs/en/deploy-sensor-map.md [hunk 10]
```diff
@@ -0,0 +1,107 @@
...
+      lng: "",
+    },
+  },
+  SHOW_MESSAGES: true,
+};
+```
+
+Here you have to insert latitude and longitude of your city.       
+
+Optionally, I can setup ["Wind direction" service](https://github.com/danwild/wind-js-server) and provide URL to it in `WIND_PROVIDER` field.
```
[the remark this answers is not in the document]
Comment (nakata5321) [reply docs/en/deploy-sensor-map.md]: solve in https://github.com/airalab/robonomics-wiki/pull/236/commits/58285cacfd4c7308b14a9952502745f80e082cc4

File: docs/en/roseman.md [hunk 11]
```diff
@@ -0,0 +1,86 @@
...
+
+RoSeMAN is Robonomics Sensors Measure Analytics and Archive Node. It is needed for accumulating sensors 
+data to show measurement history.
+
+## Requirements
+
+First, RoSeMAN requires **Mongodb** [Database server.](https://www.mongodb.com/docs/manual/introduction/) 
+In this article the installation process isn't discussed, it is assumed that you already have it. 
+
+Also, I have to turn on "Datalog" option for Sensors Connectivity module, as shown in "Scenario #3" in ["Configuration Options Description" article.](/docs/configuration-options-description/)
```
[the remark this answers is not in the document]
Comment (nakata5321) [reply docs/en/roseman.md]: solve in https://github.com/airalab/robonomics-wiki/pull/236/commits/58285cacfd4c7308b14a9952502745f80e082cc4

File: docs/en/sensor-hardware.md [hunk 12]
```diff
@@ -0,0 +1,92 @@
+---
+title: Sensor Hardware
+
+contributors: [nakata, Ludovich88]
+translated: true
+---
+
+If you wish to participate in the  air monitoring with Robonomics Open Sensors Network you need to obtain an air pollution board with sensors. 
+There are two ways to do it: 
+- order all necessary parts and build your custom board.
```
[the remark this answers is not in the document]
Comment (nakata5321) [reply docs/en/sensor-hardware.md]: solve in https://github.com/airalab/robonomics-wiki/pull/236/commits/58285cacfd4c7308b14a9952502745f80e082cc4

File: docs/en/sensors-connectivity-setup.md (hunk 1 above)
Comment (nakata5321) [reply docs/en/sensors-connectivity-setup.md]: solve in https://github.com/airalab/robonomics-wiki/pull/236/commits/58285cacfd4c7308b14a9952502745f80e082cc4

File: docs/en/configuration-options-description.md [hunk 13]
```diff
@@ -161,74 +86,23 @@ Connect you SDS011 sensor to a USB port, let's assume it got `/dev/ttyUSB0` addr
...
-> Do not forget to open the port in system firewall
->
-> On NixOS you can do:
-> ```
-> networking.firewall.allowedTCPPorts = [ 31313 ];
-> ```
+Assume that you already have MQTT broker [mosquitto](https://mosquitto.org/download/) or similar.
 
-## Scenario #3: Connect SDS011 via MQTT
+As in previous scenario you need to create new configuration file, or edit existing configuration file from previous [article](/docs/sensors-connectivity-setup/#json-configuration).
```
[the remark this answers is not in the document]
Comment (nakata5321) [reply docs/en/configuration-options-description.md]: solve in https://github.com/airalab/robonomics-wiki/pull/236/commits/58285cacfd4c7308b14a9952502745f80e082cc4

File: docs/en/configuration-options-description.md [hunk 14]
```diff
@@ -5,108 +5,33 @@ contributors: [LoSk-p, Vourhey, tubleronchik]
...
-   }
-}
-```
-At the moment it's possible to publish data to [Luftdaten](https://luftdaten.info/), [Robonomics Network](https://robonomics.network/) and [Datalog](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fkusama.rpc.robonomics.network%2F#/explorer).
-The last one is experimental!
-
-> DO NOT edit `config/default.json` file. Instead make a copy
-
-Play around with the configuration!
+This Article contains advanced configurations scenarios. If you didn't read previous article ["Sensors Connectivity Module Setup"](/docs/sensors-connectivity-setup/) first read it.
```
[the remark this answers is not in the document]
Comment (nakata5321) [reply docs/en/configuration-options-description.md:15]: Will fix in next commit.

File: docs/en/configuration-options-description.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (Vourhey) [reply docs/en/configuration-options-description.md:163]: Let's make it explicit then. 

"In this scenario it doesn't matter how data is being gathered: over HTTP, MQTT or COM. Let's look at the default HTTP configuration"

[the remark this answers is not in the document]
Comment (nakata5321) [reply docs/en/configuration-options-description.md]: solve in https://github.com/airalab/robonomics-wiki/pull/236/commits/91af83b6743445a3f314f4b315a70e7970bbf0ac

File: docs/en/configuration-options-description.md (hunk 14 above)
[the remark this answers is not in the document]
Comment (nakata5321) [reply docs/en/configuration-options-description.md]: solve in https://github.com/airalab/robonomics-wiki/pull/236/commits/91af83b6743445a3f314f4b315a70e7970bbf0ac

**Pull request merged**